MIGRATING_TO_V8.0.0.MD

```diff
@@ -0,0 +1,133 @@
+- Before 8.0.0: `rqlBuilder().eq('diagnosis', encodeURIComponent('Hypertension - STAGE 1'))`
+- After 8.0.0: `rqlBuilder().eq('diagnosis', 'Hypertension - STAGE 1')`
+
+Any use of an rql builder that does not need to support special characters may remain as it is.
+
+### Reverting double encoding for a single rql builder
+You may disable double encoding for an rql builder by setting the option `rqlBuilder({ doubleEncodeValues: false })`
+- Before 8.0.0: `rqlBuilder().eq('diagnosis', encodeURIComponent('Hypertension - STAGE 1'))`
+- After 8.0.0: `rqlBuilder({ doubleEncodeValues: false }).eq('diagnosis', encodeURIComponent('Hypertension - STAGE 1'))`
+
+### Reverting double encoding for all rql builders
+The automatic double encoding of values can be reverted to behavior before 8.0.0 using the snippet `rqlBuilder.doubleEncodeValues = false`. This should normally exist alongside the declaration of your Extra Horizon SDK instance at the root of your application.
+- You may then enable double encoding for an rql builder `rqlBuilder({ doubleEncodeValues: true }).eq('diagnosis', 'Hypertension - STAGE 1')`
+- You may manually encode values for an rql builder `rqlBuilder().eq('diagnosis', encodeURIComponent('Hypertension - STAGE 1'))`
+
+For more information regarding manual double encoding please refer to the [RQL documentation](https://docs.extrahorizon.com/extrahorizon/additional-resources/resource-query-language-rql#double-encoding-of-special-characters).
+
+## Disabling custom key normalization
+
+### Changes required in requests
+
+For all affected properties with the request mentioned, check if you use any property names that are in `camelCase` and change them to `snake_case`.
+
+Affected properties in requests had their keys transformed from camelCase to snake_case before sending it to the service.
+An example would be all properties in the `customFields` property of the profile service. Updating this field with a value of
+`{ 'helloWorld': 'test' }` would result in the value being stored as `{ 'hello_world': 'test' }`. 
+
+### Changes required in  responses
+
+For all affected properties with the response mentioned, check if you use any property names that are in `snake_case` and start addressing them as `snake_case` instead of `camelCase`.
+This includes all properties that were normalized in the requests.
+
+For all affected properties with the response mentioned, check if you use any property as if it is a date. If you do, transform it to a Date (`new Date(property)`) before using it. 
+
+Affected properties in responses had their keys transformed from snake_case to camelCase before returning the data.
+An example would be all properties in the `data` property of the configurations service. Returning a field with a value of
+`{ 'hello_world': 'test' }` would result in the value being retrieved as `{ 'helloWorld': 'test' }`.
```

Comment:
I feel like we're mixing "what is the problem" and "what changes" and "what are your options". It looks like there is a lot to explain here 🤔 I don't have a solution yet. And I do really like the information that is here, might be just a bit dense

Reply:
I tried, with an emphasis on tried to follow following structure 😅  

- State how they should change their code
- Explain why and what exactly changed 
- Give a small example